cmd/kubeadm/app/cmd/reset.go

```diff
@@ -215,11 +223,16 @@ func (r *resetData) Cfg() *kubeadmapi.InitConfiguration {
 	return r.cfg
 }
 
-// DryRun returns the DryRun flag.
+// DryRun returns the dryRun flag.
```

Comment:
this is irrelevant, but I'd like to clean this up, pls let me know whether you want to revert this back.